### PR TITLE
Database Change

### PR DESCRIPTION
**Pull request by Sanjay1611**
Signed-off-by: Sanjay Nathani <sanjay.nathani@mayadata.io>

This PR does following -

- Remove `IsEmailVerified` field
- Add `UnverifiedEmail` field

### CODE REVIEW

File: pkg/models/user.go [hunk 1]
```diff
@@ -158,12 +158,12 @@ func (u *UserCredentials) GetEmail() string {
 	return *u.Email
 }
 
-// GetIsEmailVerified user password
-func (u *UserCredentials) GetIsEmailVerified() bool {
-	if u == nil || u.IsEmailVerified == nil {
-		return false
+// GetUnverifiedEmail gets the email that user want to be verified
```
Comment (avishnu) [inline pkg/models/user.go:161]: that needs to be verified

File: pkg/oauth/providers/github.go [hunk 2]
```diff
@@ -44,7 +44,6 @@ func getUserFromToken(c *gin.Context, token *oauth2.Token) (*models.UserCredenti
 	for _, githubUserEmail := range githubUserEmails {
 		if *githubUserEmail.Primary == true {
 			user.Email = githubUserEmail.Email
```
Comment (avishnu) [inline pkg/oauth/providers/github.go:46]: Do we need to set UnverifiedEmail = nil or ""?

Comment (Sanjay1611) [reply pkg/oauth/providers/github.go:46]: It will automatically be set to nil until we set it othewise

File: pkg/models/user.go [hunk 3]
```diff
@@ -158,12 +158,12 @@ func (u *UserCredentials) GetEmail() string {
 	return *u.Email
 }
```
Comment (avishnu) [inline pkg/models/user.go:160]: that needs to be verified*

File: pkg/models/user.go [hunk 4]
```diff
@@ -158,12 +158,12 @@ func (u *UserCredentials) GetEmail() string {
...
 
-// GetIsEmailVerified user password
-func (u *UserCredentials) GetIsEmailVerified() bool {
-	if u == nil || u.IsEmailVerified == nil {
-		return false
+// GetUnverifiedEmail gets the email that user needs to be verified
```
Comment (avishnu) [inline pkg/models/user.go:161]: that needs to be verified*
Please remove the word "user"

**Review by avishnu — APPROVED** [review]
lgtm

**Pull request merged**